qiling/os/os.py

```diff
@@ -92,32 +92,32 @@ def find_containing_image(self, pc):
                 return image
 
     def emu_error(self):
-        logging.info("\n")
+        logging.error("\n")
 
         for reg in self.ql.reg.register_mapping:
             if isinstance(reg, str):
                 REG_NAME = reg
                 REG_VAL = self.ql.reg.read(reg)
-                logging.info("[-] %s\t:\t 0x%x" % (REG_NAME, REG_VAL))
+                logging.error("%s\t:\t 0x%x" % (REG_NAME, REG_VAL))
```

Comment:
I removed the prefix here, since the `logging` formatter already puts a prefix like this in front.
I assumed that this was a leftover from a global replace with the `ql.nprint` to `logging.info` ?